bin/index.js

```diff
@@ -11,13 +11,15 @@ const config = require('./config.js');
 program
   .version('0.0.1')
   .option('-b, --bundle-name [filename]', 'Output bundle filename')
+  .option('-p, --package-json [path]', 'path to package.json directory with react-build-dist config override')
```

Comment:
Just for consistency's sake, maybe it's best to capitalize the `P` in the description:

```
'Path to package.json directory...
```